geomstats/geometry/special_orthogonal.py

```diff
@@ -81,6 +81,54 @@ def random_uniform(self, n_samples=1, tol=1e-6):
         skew = self.to_tangent(random_mat)
         return self.exp(skew)
 
+    @staticmethod
+    def angle_of_rot2(r):
+        """
+        Get angle of a 2D rotation.
+
+        :param r: array-like, shape=[n, n]: rotation
+        :return: float: angle of rotation
+        """
+        # assert (self.n == 2),\
+        #     'This function was only tested for n==2, do not use otherwise.'
+        return gs.arctan(r[0][1] / r[0][0])
+
+    def divide_angle_of_cov2(self, r, alpha):
```

Comment:
The method is called `divide`, but it multiplies the angle by alpha?

Maybe `multiply_by_angle_2d` (to also get rid of `cov2` which is too specific for the more general SpecialOrthogonal class)?

Reply:
Renamed to `multiply_angle_of_rot2`; as is, it is specific to a 2D rotation matrix.